### PR TITLE
fixing the docstring for one of the test in pbs_offline_vnodes.py

### PR DESCRIPTION
**Pull request by anamikau**
#### Bug/feature Description

* *Bugs: doc string for one of the test was not accurate*

#### Affected Platform(s)
* *Platform (None)*

#### Cause / Analysis / Design
* *Bugs: docstring has additional content that doesn't belonged to that test*

#### Solution Description
* *updated the docstring*

#### Testing logs/output
* * 
[docstringfix.txt](https://github.com/PBSPro/pbspro/files/2702088/docstringfix.txt)
 *

#### Checklist:

- [x] I have joined the **[pbspro community forum](http://community.pbspro.org/)**.
- [x] My pull request contains a **single, signed** commit. See **[setting up gpg signature](https://pbspro.atlassian.net/wiki/display/DG/Signing+Your+Git+Commits)**.
- [x] My code follows the **[coding style](https://pbspro.atlassian.net/wiki/display/DG/Coding+Standards)** of this project.
- [ ] My change requires project documentation. See **[required documentation checklist](https://pbspro.atlassian.net/wiki/display/DG/Checklist+for+Developing+Features+and+Bug+Fixes)** for details.
   - [ ] I have added documentation in the **[project documentation area](https://pbspro.atlassian.net/wiki/display/PD)**.
- [ ] I have added new **PTL test(s) to my commit**. (See **[using PTL for testing](https://pbspro.atlassian.net/wiki/display/DG/Using+PTL+for+Testing)**) *(or)*
   - [ ] I have added  **manual test(s) to this pull request and explained why PTL is not appropriate** for this case.
- [ ] All new and existing automated tests have passed. (See **[running automated PTL tests](https://pbspro.atlassian.net/wiki/display/DG/PTL+Quick+Start+Guide)**).
- [ ] I have attached **test logs to this pull request** as evidence of testing/verification.

__***For further information please visit the [Developer Guide Home](https://pbspro.atlassian.net/wiki/display/DG/Developer+Guide+Home).***__

### CODE REVIEW

File: test/tests/functional/pbs_offline_vnodes.py [hunk 1]
```diff
@@ -273,9 +273,6 @@ def test_multi_mom_hook_failure_affects_vnode2(self):
         when executed by sister mom, causing
         the server to execute the fail_action=offline_vnodes, which
         result in sister vnode to be marked offline.
-        Then run an exechost_startup hook that gets an
-        exception when local mom is restarted. Vnode representing
-        local mom would be marked offline.
```
Comment (mike0042) [inline test/tests/functional/pbs_offline_vnodes.py]: Indentation appears off by one space in some of the surrounding lines.

Comment (anamikau) [reply test/tests/functional/pbs_offline_vnodes.py]: Hey @mike0042 I ran pycodestyle and I do not see anything. Travis is also not complaining about that.

camac11:functional anamika$ pycodestyle pbs_offline_vnodes.py 
camac11:functional anamika$ echo $?
0

Comment (mike0042) [reply test/tests/functional/pbs_offline_vnodes.py]: I looked at the raw version of the file, and it looks fine. It's just the way GitHub is displaying the diff. Thanks for confirming.

**Comment by mike0042** [comment]
AppVeyor is having problems again. Hopefully, it will resolve itself once you rebase for merge.

**Comment by anamikau** [comment]
Thanks @mike0042 . Yes I will rebase soon.

**Comment by anamikau** [comment]
I have rebased but waiting for another review.

**Pull request merged**